appveyor.yml

```diff
@@ -13,7 +13,7 @@ configuration:
 before_build:
   - git submodule update --init --recursive
   - python get_steinberg_sdk.py
-  - cmake -G "Visual Studio 14" Smexoscope
+  - cmake -G "Visual Studio 14 2015 Win64" .
```

Comment:
p.s. I don't think it is static linking to the msvc run times. I will need to look into this. Typically I don't like doing this but I know there a lot of VST users that are allegric to run times for some reason. :smile: